hack/gen-client-python/deploy.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+set -e
+
+source $(dirname "$0")/../../hack/common.sh
+
+cd ${PYTHON_CLIENT_OUT_DIR}
+bash git_push.sh
```

Comment:
I am afraid that this generated script is not suitable enough for us.
It doesn't remove files which were removed by change in API, so it will lead to old files accumulation.

Reply:
you could just delete the whole content (except .git) before copying over the files. that should solve that.

Reply:
The thing is that script initialize repository directly inside of output-directory.
https://gist.github.com/lukas-bednar/b3f6b749efd983b9b81630589177106b
Maybe I can remove `git init` & `git remote add` lines from that script.

Reply:
btw. is the git_push.sh script missing in the PR?

Reply:
I would just split the script in git_prepare and git_push and delete the content in between.

Reply:
> btw. is the git_push.sh script missing in the PR?
it is generated by swagger-code-gen.

Reply:
@rmohr I decided to do not use this auto-generated git_push.sh script.
* I already have several sed commands to alter it (token name, adding tags, pushing tags)
* it doesn't clean up files which were removed
* it fails on git-pull command without `--allow-unrelated-histories` (another sed command)
* git-pull fail to merge *.log file because they are in .gitignore (another sed)

it seems like that I am writing script which rewrites other script to fit our needs.